backend/app/api/endpoints/runtime_work.py

```
@@ -824,6 +825,25 @@ async def create_runtime_task_endpoint(
     )
 
 
+@router.post(
+    "/team/create",
+    response_model=RuntimeTaskCreateResponse,
+    response_model_by_alias=True,
+)
+async def create_team_runtime_task_endpoint(
+    request: RuntimeTeamTaskCreateRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.16.3)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)